jupyterlab-generic/jupyterlab-start.sh

```diff
@@ -1,21 +1,21 @@
 #!/usr/bin/env bash
 
-# Symlink config files for persistence
-test -f ./pd/.bash_profile || touch ./pd/.bash_profile
-test -f ./pd/.bashrc || touch ./pd/.bashrc
-test -d ./pd/.jupyter || mkdir ./pd/.jupyter
-test -d ./pd/.ipython || mkdir ./pd/.ipython
-test -d ./pd/.config || mkdir ./pd/.config
-test -d ./pd/.local || mkdir ./pd/.local
-ln -s ./pd/.bash_profile .
-ln -s ./pd/.bashrc .
-ln -s ./pd/.jupyter .
-ln -s ./pd/.ipython .
-ln -s ./pd/.config .
-ln -s ./pd/.local .
+# # Symlink config files for persistence
+# test -f ./pd/.bash_profile || touch ./pd/.bash_profile
+# test -f ./pd/.bashrc || touch ./pd/.bashrc
+# test -d ./pd/.jupyter || mkdir ./pd/.jupyter
+# test -d ./pd/.ipython || mkdir ./pd/.ipython
+# test -d ./pd/.config || mkdir ./pd/.config
+# test -d ./pd/.local || mkdir ./pd/.local
+# ln -s ./pd/.bash_profile .
+# ln -s ./pd/.bashrc .
+# ln -s ./pd/.jupyter .
+# ln -s ./pd/.ipython .
+# ln -s ./pd/.config .
+# ln -s ./pd/.local .
 
-# Symlink data directory mounted at root
-ln -s /data .
+# # Symlink data directory mounted at root
+# ln -s /data .
```

Comment:
I think we should remove this, since this is already being done by hatchery
those config file symlinks could stay commented above, since we still need to figure out exactly how could we address them